beacon_node/eth1/src/http.rs

```
@@ -282,7 +324,7 @@ async fn call(
     ]);
 
     let response_body = send_rpc_request(endpoint, "eth_call", params, timeout).await?;
-    match response_result(&response_body)? {
+    match response_result_or_error(&response_body).ok() {
```

We're suppressing errors here.

Previously, if the server returned an `error` in the response we'd exit early with `Err`. Now, we return `Ok(None)`, indicating the server successfully returned with no data.